apps/client/src/pages/ResultsPage.js

```diff
@@ -38,6 +41,12 @@ const ResultsPage = ({ topic, checker }) => {
         }}
         data-testid={RESULTS_PAGE}
       >
+        <Paragraph>
+          Vergunningcheck uitgevoerd voor: <AddressLine address={address} />
+        </Paragraph>
+
+        <AddressData displayZoningPlans={false} address={address} />
```

Comment:
`displayZoningPlans={false}` is default so this can be removed

Reply:
true.